verifier/verifier.py

```diff
@@ -106,16 +105,11 @@ def pre_verify(self, source, rendition):
 
     def meta_model(self, row):
         """
-        Inputs the metamodel AND operator as condition
-        Retrieves the tamper value of the UL model only when both models agree in classifying
-        as non tampered. Otherwise retrieves the SL classification
-        UL classifier has a higher TPR but lower TNR, meaning it is less restrictive towards
-        tampered assets. SL classifier has higher TNR but is too punitive, which is undesirable,
-        plus it requires labeled data.
+        The goal is to reduce the number of False Positives (tamper) to prevent wrongfully penalizing transcoder nodes. OCSVM model is expected to have higher precision (low FP) on novel data.
+        If OCSVM predicts the observation is an inlier (not tampered), we'll go with it, otherwise we'll use supervised model output.
         """
-        meta_condition = row['ul_pred_tamper'] == 1 and row['sl_pred_tamper'] == 1
-        if meta_condition:
-            return row['ul_pred_tamper']
+        if row['ul_pred_tamper'] == 0:
```

Comment:
Why the change in the meta model condition?

Reply:
I believe it was incorrect before, because of confusion of what's the return of OCSVM model in case it predicts tamper (it returns -1 for an outlier, and 1 for inlier). The prediction is now converted to match 1=Tamper notation. Regarding the specific condition, we trust OCSVM model, if it detects an outlier, and use supervised Catboost model in all other cases. It gives additional 0.01 precision on test data.
From the comment:
        The goal is to reduce the number of False Positives (tamper) to prevent wrongfully penalizing transcoder nodes. OCSVM model is expected to have higher precision (low FP) on novel data.
        If OCSVM predicts the observation is an inlier (not tampered), we'll go with it, otherwise we'll use supervised model output.